### PR TITLE
Add chart support to MooseDocs

### PR DESCRIPTION
**Pull request by aeslaughter**

### CODE REVIEW

File: framework/doc/content/utilities/MooseDocs/extensions/plotly.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+A complete list of settings available to the `plot scatter` command are included in
+[plot-scatter-settings].
+
+!devel settings module=MooseDocs.extensions.plotly
+                object=PlotlyScatter
+                id=plot-scatter-settings
+                caption=Available settings for the `plot scatter` command.
+
+[plotly]: https://plot.ly/
```
Comment (sapitts) [inline framework/doc/content/utilities/MooseDocs/extensions/plotly.md]: would you give a direct link to the plotly manual?  I'm having trouble finding my way past the pricing information to find a users manual

Comment (sapitts) [reply framework/doc/content/utilities/MooseDocs/extensions/plotly.md]: Nevermind, I completely missed the link to the reference manual above

**Review by sapitts — APPROVED** [review]
Looks like good docs to me -- I'm looking forward to trying out this capability

**Comment by sapitts** [comment]
Will the matplotlib plugin work with the javascript version of plotly? https://plot.ly/matplotlib/

**Review by brianmoose — APPROVED** [review]

**Pull request merged**